Library/ViewModels/ProjectPamphletMainCellViewModelTests.swift

```diff
@@ -623,44 +629,16 @@ final class ProjectPamphletMainCellViewModelTests: TestCase {
       XCTAssertEqual(self.optimizelyClient.trackedAttributes?["session_device_type"] as? String, "phone")
     }
   }
+}
 
-  func testTrackingProjectViewed_LiveProject_LoggedIn_NonBacked() {
```

Comment:
Why was this one deleted?

Reply:
Because Project Page Viewed event for campaign context was moved away from the `ProjectPamphletMainCellViewModel`.